editor/modes/visual-editor/style.scss

```diff
@@ -83,6 +82,76 @@
 		transition: 0.2s outline;
 	}
 
+	&.is-collaboration {
```

Comment:
This class should be more semantic: `is-being-edited` perhaps.

Reply:
Updated.